wordpress-mobile/WordPress-iOS
# [Design System] Update color values to match Figma

Refs #p1702909565644119/1702665510.828399-slack-C04TXQU42HM
Figma File: LyzJJa6ANKToUswWmRrZee-fi-102_186

## Testing Steps:
1. Install and login Jetpack
2. Navigate to "Design System" and check if Secondary, Tertiary and Quaternary background color values match the designs on both themes.
3. Make sure "Divider" color is removed from Color.xcassets in Design System Module

## Regression Notes
1. Potential unintended areas of impact
The callsites for the changed values.

4. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested them.

5. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


UI Changes testing checklist:
